src/components/Table/TaskLink.tsx

```diff
@@ -1,17 +1,24 @@
+import { Body } from "@leafygreen-ui/typography";
 import { StyledRouterLink, WordBreak } from "components/styles";
 import { getTaskRoute } from "constants/routes";
+import { executionAsDisplay } from "pages/task/util/execution";
```

Comment:
Should we move this to a more general location so we aren't importing from page directories?

perhaps "utils/task"